index.html

```diff
@@ -1,57 +1,226 @@
-        <label for="box2">Box2
-      <input id="box2" type="checkbox" name="box2" value="box2">
-    </form>
-
-    <div id= "comment-section">
-      <label for="comments">
-        Do you have any Comments?</label>
-
-        <textarea id="comments" class="input-field" style="height:50px;resize:vertical;" name="comment" placeholder="Enter your comment here..."></textarea>
-        </div>
-        <div><button type="submit" id="submit" class="dropdown"> Submit </button></div>
-
-        <script src="js/index.js"></script>
-</body>
-
+  <head>
+    <meta charset="utf-8">
+    <title>Home</title>
+    <link href="css/bootstrap.css" rel="stylesheet">
+    <link href="css.css" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container">
+      <header class="header">
+        <h1 id="title" class="text-center">Meditation app survey form</h1>
+        <p id="description" class="description text-center">
+          Thank you for taking the time to help us improve our app
+        </p>
+      </header>
+      <form id="survey-form">
+        <div class="form-group">
+          <label id="name-label" for="name">Name</label>
+          <input
+            type="text"
+            name="name"
+            id="name"
+            class="form-control"
+            placeholder="Enter your name"
+            required
+          />
+        </div>
+        <div class="form-group">
+          <label id="email-label" for="email">Email</label>
+          <input
+            type="email"
+            name="email"
+            id="email"
+            class="form-control"
+            placeholder="Enter your Email"
+            required
+          />
+        </div>
+        <div class="form-group">
+          <label id="number-label" for="number"
+            >Age<span class="clue">(optional)</span></label
+          >
+          <input
+            type="number"
+            name="age"
+            id="number"
+            min="10"
+            max="99"
+            class="form-control"
+            placeholder="Age"
+          />
+        </div>
+        <div class="form-group">
+          <p>Which option best describes your current role?</p>
+          <select id="dropdown" name="role" class="form-control" required>
+            <option disabled selected value>Select current role</option>
+            <option value="student">Student</option>
+            <option value="job">Full Time Job</option>
+            <option value="unemployed">Unemployed</option>
+            <option value="preferNo">Prefer not to say</option>
+            <option value="other">Other</option>
+          </select>
+        </div>
+    
+        <div class="form-group">
+          <p>Would you recommend our meditation app to a friend?</p>
+          <label>
+            <input
+              name="user-recommend"
+              value="definitely"
+              type="radio"
+              class="input-radio"
+              checked
+            />Definitely</label
+          >
```

Comment:
Kindly close the tags within one line if possible to enhance a good visual appeal when viewers are going through your code. Example: `<label id="id-name" for="purpose">GPA<span class="class-name">4.0</span></label>`